pcm_dynamic_jams/demo/data/scripts/class.lua

```diff
@@ -0,0 +1,69 @@
+---------------------------------------------------------------
+-------- functions for OOP              --------
```

Comment:
yes, iirc it's required for the demo so I felt it appropriate to include it in the demo. Otherwise anyone attempting to test the demo would need to figure out and add the dependencies used by the mission

Reply:
> Otherwise anyone attempting to test the demo would need to figure out and add the dependencies used by the mission

You can assume that anyone using this repository would know how to do that.  The entire point of specifying a dependency is to avoid duplicating files in every place that uses them.

Look at PromptBox for example.  It depends on AxBase and it will not function without the files in AxBase.  Yet the PromptBox demo does not come bundled with the AxBase files.